### PR TITLE
#89218 - validate params in vars

### PR DESCRIPTION
**Pull request by mszymeshop**

### CODE REVIEW

File: src/Platform.Eda.Cli/Commands/ConfigureEda/JsonProcessor/PutSubscriberProcessChain.cs [hunk 1]
```diff
@@ -96,7 +96,6 @@ public async Task<int> ProcessAsync(string inputFolderPath, string env, Dictiona
                 {
                     _console.WriteNormal("Extracting variables");
                     varsJObject = (JObject)parsedFile["vars"];
-                    parsedFile.Remove("vars");
```
Comment (mszymeshop) [inline src/Platform.Eda.Cli/Commands/ConfigureEda/JsonProcessor/PutSubscriberProcessChain.cs:99]: if we remove, we cannot validate if params are in vars

**Review by tabman83 — APPROVED** [review]
just don't understand what changed around regexes

**Review by nikhilgirraj — APPROVED** [review]

**Comment by tabman83** [comment]
I expect unit tests to fail here but my pr off this branch fixes them

**Comment by mszymeshop** [comment]
> I expect unit tests to fail here but my pr off this branch fixes them

Nino, I have fixed the tests now.

**Pull request merged**